LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
 
-**Derivative**: Software which includes any portion of the Mod other than the API, with modifications, additions, or subtractions.
+**Derivative**: Software which includes at least some portion of the Mod other than the API.
```

Comment:
There was a technical contradiction with the old statement. Under the old wording, a "Derivative" technically included the empty set. That is, take the mod and then delete everything. Clearly, that's not a derivative, but the old wording includes it.